main.go

```diff
@@ -113,6 +119,28 @@ func main() {
 		runMapper = mapper.Run
 		stopMapper = mapper.Stop
 	}
+	if flagExecNodeHost != "" {
+		feeder, err := feeder.FromHost(log, flagExecNodeHost)
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not initialize feeder")
+		}
+
+		chain, err := chain.FromHost(log, flagExecNodeHost)
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not initialize chain")
+		}
```

Comment:
What would be the potential advantages to get them from the access nodes instead? I was also thinking that maybe linking up to as many nodes as possible would actually be nice, since this way the DPS would be more resilient to network issues.

For example, we could subscribe to the trie updates of all exec nodes that expose their socket, and if one of them misses an update but another sends it, we're fine and we don't need to send a request. WDYT?

Maybe then it would make sense to also connect to access nodes to increase the resilience even further.